src/server.rs

```diff
@@ -553,3 +650,23 @@ impl WalletService for Arc<crate::wallet::Wallet> {
         ))
     }
 }
+
+fn bdk_block_hash_to_bitcoin_block_hash(hash: bdk::bitcoin::BlockHash) -> bitcoin::BlockHash {
```

Comment:
Nit: out-of-order declarations are very uncomfortable to read. Rust functions/blocks read top-to-bottom, so should modules unless there is a good reason not to